### PR TITLE
hips: fix enabling E2EE for tracks added later to the conference

### PR DESCRIPTION
**Pull request by saghul**

### CODE REVIEW

File: modules/e2ee/E2EEContext.js [hunk 1]
```diff
@@ -31,6 +31,9 @@ const unencryptedBytes = {
     undefined: 1 // frame.type is not set on audio
 };
 
+// Flag to set on senders / receivers to avoid setting up the encryption transform
+// more than once.
+const kJitsiE2EE = Symbol('kJitsiE2EE');
```
Comment (fippo) [inline modules/e2ee/E2EEContext.js:36]: oh I didn't know the symbol trick!

File: JitsiConference.js [hunk 2]
```diff
@@ -3327,3 +3313,43 @@ JitsiConference.prototype.setE2EEKey = function(key) {
...
+/**
+ * Setup E2EE for the sending side, if supported.
+ * Note that this is only done for the JVB Peer Connecction.
+ *
+ * @returns {void}
+ */
+JitsiConference.prototype._setupSenderE2EEForTrack = function(track) {
+    const jvbPc = this.jvbJingleSession ? this.jvbJingleSession.peerconnection : null;
+
+    if (!this.isP2PActive() && jvbPc && this._e2eeCtx) {
```
Comment (paweldomas) [inline JitsiConference.js:3326]: do you really  need the `isP2PActive()` check?

Comment (paweldomas) [reply JitsiConference.js:3326]: I suppose you don't because you're working with the JVB peerconnection anyway

Comment (saghul) [reply JitsiConference.js:3326]: Right, I could simplify!

**Comment by saghul** [comment]
Updated, PTAL @fippo @paweldomas

**Pull request merged**